client/src/pages/course/admin/stage-interviews.tsx

```diff
@@ -46,17 +46,22 @@ function Page(props: CoursePageProps) {
       courses={[props.course]}
     >
       <Row style={{ marginBottom: 16 }} justify="space-between">
-        <Button type="primary" onClick={() => setModal(true)}>
-          Create
-        </Button>
         {courseManagerRole ? (
           <div>
             <Checkbox checked={noRegistration} onChange={e => setNoRegistration(e.target.checked)}>
               No Registration
             </Checkbox>
-            <Button onClick={createInterviews}>Create Interviews</Button>
+            <Popconfirm
+              onConfirm={() => createInterviews()}
+              title="Do you want to create interview pairs for not distribute students?"
```

Comment:
is it about not distributed yet students? then you should say `distributed`

Reply:
fixed